java/Oleksii-Riabov/src/com/zoolatech/lecture5/tasks/_5/Task5.java

```diff
@@ -0,0 +1,60 @@
+package com.zoolatech.lecture5.tasks._5;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.PriorityQueue;
+
+/**
+ * Create a class KthLargest with a single method int add(int value) that
+ * accepts a value and returns the kth largest value in a stream of numbers
+ * (the amount of numbers is increasing). Number k should be passed during
+ * the object construction. Return -1 if the number of values seen so far
+ * is less than k.
+ * new KthLargest(2)
+ * add(1) // -1
+ * add(2) // 1
+ * add(5) // 2
+ * add(6) // 5
+ * add(4) // 5
+ */
+
+public class Task5 {
+
+    public static void main(String[] args) {
+        KthLargest kthLargest = new KthLargest(2);
+        System.out.println(kthLargest.add(1));
+        System.out.println(kthLargest.add(2));
+        System.out.println(kthLargest.add(5));
+        System.out.println(kthLargest.add(6));
+        System.out.println(kthLargest.add(4));
+    }
+
+    static class KthLargest {
+
+        private final int k;
+        private final PriorityQueue<Integer> min = new PriorityQueue<>();
+
+        public KthLargest(int k) {
+            this.k = k;
+        }
+
+        public List<Integer> add(int value) {
+            List<Integer> list = new ArrayList<>();
+            if(min.size() < k) {
```

Comment:
```
if(min.size() < k)
```
```
Return -1 if the number of values seen so far is less than k.
```

Reply:
Method "add" have been updated to immediately return -1 if "value" less than "k"